apps/ledger-live-desktop/src/renderer/components/DeviceIllustration.tsx

```diff
@@ -51,6 +51,11 @@ const illustrations: { [key in DeviceModelId]: Illustration } = {
     width: 53.83,
     height: 250.87,
   },
+  nanoFTS: {
+    Illustration: NanoX,
```

Comment:
Maybe we can use FTS placeholders from Figma : https://www.figma.com/file/mntfjbMo3J9S0wbYfFPSoI/FTS-%2F-Onboarding

Reply:
Nope we can't